airflow/models/variable.py

```diff
@@ -131,6 +133,18 @@ def get(
         :param default_var: Default value of the Variable if the Variable doesn't exist
         :param deserialize_json: Deserialize the value to a Python dict
         """
+        try:
```

Comment:
could also just fire an analytics event

Reply:
does this mean logs? i figured it might be easier to analyze results using sql (in pr description) if we have a table